src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -183,6 +185,25 @@ else if (!TokenUtil.isOfType(currentType, IGNORED_LIST)) {
         }
     }
 
+    /**
+     * Checks whether the method lparen starts from a new line or parameter node has any child
```

Comment:
Please split this method into two separated methods: one to to check that the method lparen starts from a new line, another for childless `PARAMETERS` node. They can be simple `return ...` without any `if`.

Reply:
@pbludov done. There is one problem I am facing ,the cyclometric complexity of  

`checkIndentation(DetailAST firstNode, DetailAST lastNode, int indentLevel, int startIndent, LineWrappingOptions ignoreFirstLine)` is increasing from 10 to 11 because of the second branch in 
`if(checkForNullParameterChild(node) || checkForMethodLparenNewLine(node))`. I have pushed the code to let you see.
I think another  method has to be created returning the value of the expression inside if.

Reply:
Better to refactor `checkIndentation` method a bit. For example, Extract lines 143-157 into a dedicated method.

Reply:
Done. Sir I am not getting why travis/ci is failing. There is some issue with processing pom.
Could you please help me out with this issue?